config/ci.mocha.node.opts

```diff
@@ -0,0 +1,8 @@
+# With dot reporter, the full names of test case will not be displayed in log during execution.
+# The name and details of all failed cases will still be appended to log after execution.
+# See https://mochajs.org/#dot-matrix for more information.
+--reporter dot
+
+# Node doesn't exit after mocha test completion. Supplying this flag for force exit.
+# Not applicable in karma + mocha test (browser test).
+--exit
```

Comment:
Am I confused or do we no longer set a timeout / retries in CI?

I'm not opposed to playing with changing these, but I'd want to do it in a separate PR since I think it's orthogonal to the logging output (well, mostly).

Reply:
This is the mocha config for node testing.

I think it makes sense to separate out the change for mocha config file. I though it would be simple and straightforward at first, but it does feel confusing in this PR now.